src/Test/WebsiteFullTrust/WebsiteFullTrust.csproj

```diff
@@ -147,7 +147,7 @@
       <Version>3.1.24</Version>
```

Comment:
Maybe we should update efcore 3.1 to latest as well

Reply:
Sounds good, will do